modules/serialize/src/decode_body_header.ts

```diff
@@ -148,7 +148,8 @@ export function decodeFinalFrameBodyHeader (buffer: Uint8Array, headerInfo: Head
 
   /* The precondition SEQUENCE_NUMBER_END: Uint32(FFFF) is handled above. */
   const sequenceEnd = dataView.getUint32(readPos, false) // big endian
-  needs(sequenceEnd === SequenceIdentifier.SEQUENCE_NUMBER_END, '')
+  /* Postcondition: sequenceEnd must be SEQUENCE_NUMBER_END. */
+  needs(sequenceEnd === SequenceIdentifier.SEQUENCE_NUMBER_END, 'Malformed sequenceEnd')
```

Comment:
This seems too specific and unlikely that an arbitrary developer is going to understand what it means.
```suggestion
  needs(sequenceEnd === SequenceIdentifier.SEQUENCE_NUMBER_END, 'Malformed final frame: Invalid sequence number end value')
```